iOS/Collection View Management /DiffableCollectionViewController.swift

```diff
@@ -75,7 +75,7 @@ class DiffableCollectionViewController<SectionType: Hashable,
 
         let snapshot = self.getInitialSnapshot(with: dataDictionary)
 
-        if let animationCycle = self.getAnimationCycle() {
+        if let animationCycle = self.getAnimationCycle(withData: dataDictionary) {
```

Comment:
When yo create the animation cycle, the data dictionary is also provided. This allows us to scroll to a specific message.

Reply:
I needed this.